client/sites-dashboard/components/sites-table-row/docs/example.jsx

```diff
@@ -0,0 +1,212 @@
+
+export default function SitesTableRowExample() {
+	const exampleSite = {
+		URL: 'https://wpvip.com',
+		is_coming_soon: false,
+		is_private: false,
+		visible: true,
+		launch_status: 'launched',
+		name: 'The long title of the sample site',
+		jetpack: false,
+		site_owner: 12,
+		lang: 'en',
+		options: {
+			admin_url: 'https://wpvip.com/wp-admin/',
+			is_redirect: false,
+			is_wpforteams_site: false,
+			launchpad_screen: false,
+			site_intent: '',
+			unmapped_url: 'https://wpvip.com',
+			updated_at: new Date(),
+		},
+		p2_thumbnail_elements: null,
+		plan: {
+			product_id: 1,
+			product_slug: 'free_plan',
+			product_name_short: 'Free',
+			expired: false,
+			user_is_owner: false,
+			is_free: true,
+		},
+		is_wpcom_atomic: false,
+		title: 'The long title of the sample site',
+		slug: 'wpvip.wordpress.com',
+	};
+
+	const siteSimpleFree = {
+		...exampleSite,
+		ID: 1,
+	};
+
+	const siteSimpleBusinessComingSoon = {
+		...exampleSite,
+		ID: 2,
+		options: {
+			...exampleSite.options,
+			updated_at: new Date( +new Date() - 60 * 60 * 24 * 1000 ),
+		},
+		plan: {
+			...exampleSite.plan,
+			product_id: 1008,
+			product_slug: 'business-bundle',
+			product_name_short: 'Business',
+			user_is_owner: true,
+			is_free: false,
+		},
+		is_coming_soon: true,
+		launch_status: 'unlaunched',
+	};
+
+	const siteSimpleBusinessExpired = {
+		...exampleSite,
+		ID: 3,
+		options: {
+			...exampleSite.options,
+			updated_at: new Date( +new Date() - 60 * 60 * 24 * 1000 ),
+		},
+		plan: {
+			...exampleSite.plan,
+			product_id: 1008,
+			product_slug: 'business-bundle',
+			product_name_short: 'Business',
+			user_is_owner: true,
+			is_free: false,
+			expired: true,
+		},
+	};
+
+	const siteJetpackComplete = {
+		...exampleSite,
+		ID: 4,
+		jetpack: true,
+		options: {
+			...exampleSite.options,
+			updated_at: new Date( +new Date() - 60 * 60 * 24 * 2 * 1000 ),
+		},
+		plan: {
+			...exampleSite.plan,
+			product_id: 2014,
+			product_slug: 'jetpack_complete',
+			product_name_short: 'Complete',
+			user_is_owner: true,
+			is_free: false,
+		},
+	};
+
+	const siteSimplePrivateP2 = {
+		...exampleSite,
+		ID: 5,
+		is_private: true,
+		visible: false,
+		options: {
+			...exampleSite.options,
+			is_wpforteams_site: true,
+			updated_at: new Date( +new Date() - 60 * 60 * 24 * 30 * 1000 ),
+		},
+		plan: {
+			...exampleSite.plan,
+			product_id: 1040,
+			product_slug: '"wp_p2_plus_monthly"',
+			product_name_short: 'P2+',
+			is_free: false,
+		},
+		icon: {
+			ico: 'https://wpjobmanager.com/wp-content/uploads/2022/10/wp-job-manager-logo.webp?w=16',
+			img: 'https://wpjobmanager.com/wp-content/uploads/2022/10/wp-job-manager-logo.webp',
+		},
+		p2_thumbnail_elements: {
+			color_link: '#0267ff',
+			color_sidebar_background: '#d0ccb8',
+			header_image:
+				'https://wpjobmanager.com/wp-content/uploads/2022/11/hero-resized.png?resize=680',
+		},
+	};
+
+	return (
+		<div>
+			<h2>Free Simple Site</h2>
```

Comment:
Would it be more readable if we kept all sites in one `<table>`, added `<thead>`, and used the example titles e.g. "Free Simple Site" as the site name?

Reply:
I don't have a strong opinion.
I guess it would be simpler to have a unique table.
Then we could have an array of sites:
```jsx
{ [
	siteSimpleFree,
	siteSimpleBusinessComingSoon,
	siteSimpleBusinessExpired,
	siteJetpackComplete,
	siteSimplePrivateP2,
].map( ( site ) => (
	<SitesTableRow site={ site } key={ site.ID } />
) ) }
```